de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/async/GenerateInOutFromHU.java

```diff
@@ -163,6 +165,7 @@ public InOutGenerateResult generateInOuts(final Properties ctx,
 			final String processShipmentsDocAction,
 			final boolean createPackingLines,
```

Comment:
here you could reuse the params-pojo I mentioned further up

Reply:
Talked with Tobi and decided it's not needed to reuse the pojo here because this method already uses some "workpackage parameters" from WorkpackageProcessorAdapter. It would make it harder to read if I added a new kind of them. Also, I would have to recreate the pojo and in this place I would miss the initial filters, which are mandatory.